0009-enpoint-events-ipc/README.md

```diff
@@ -0,0 +1,126 @@
+---
+name: Endpoint events IPC
+shortname: 9/ENDPOINT_EVENTS
+status: draft
+editor: Volodymyr Tkhir <vtkhir@cybervisiontech.com>
+---
+
+* [Introduction](#introduction)
+* [Connectivity events](#connectivity-events)
+  * [Endpoint connected](#endpoint-connected)
+  * [Endpoint disconnected](#endpoint-disconnected)
+* [Lifecycle events](#lifecycle-events)
+  * [Endpoint registered](#endpoint-registered)
+  * [Endpoint updated application version](#endpoint-updated-application-version)
+* [Metadata events](#metadata-events)
+  * [Endpoint metadata updated](#endpoint-metadata-updated)
+* [Configuration events](#configuration-events)
+  * [New endpoint configuration available](#new-endpoint-configuration-available)
+  * [Endpoint applied the configuration](#endpoint-applied-the-configuration)
+
+
+## Introduction
+
+The Endpoint events IPC protocol is a [Broadcast Messaging](/0003-messaging-ipc/README.md#broadcast-messaging) protocol extension.
+
+Document describes the endpoint events that are published into NATS message broker. The document describes the NATS subject and Avro message format for each message.
+The NATS subject format is:
+
+  `kaa.v1.events.{originator-service-instance-name}.endpoint.{event-group}.{event-type}`
+
+  where:
+
+  - `{originator-service-instance-name}` - the name of the originator service instance. Allows consumers to subscribe to events from a specific source.
+  - `{event-group}` - the logical group of events. For example, connectivity, lifecycle, configuration, metadata, etc.
+  - `{event-type}` - specific event type. For example, endpoint connectivity events the types will be connected, disconnected, dormant, awake, etc.
+
+## Connectivity events
+The `{event-group}` is `connectivity`.
+### Endpoint connected
+Published when endpoint opened the communication session with the Kaa Iot Platform.
```

Comment:
Should we rather say "server" instead of "Kaa IoT Platform" here? What do you think @rasendubi @hunch7.